gramineproject/gsc
# Fix `base-gramine` generated Docker image name

## Description of the changes 

Please see #181 for context

Fixes #181

## How to test this PR? 

1. Create base-Gramine Docker image without this PR and use that image with `gsc build` with this PR, build should succeed with a warning.
2. Create base-Gramine Docker image with this PR and use that image with `gsc build` with this PR, build should succeed without any warnings.
3. Specify base-Gramine docker image under Gramine tag in configuration file which doesn't exists on system and do `gsc build`, build should report an error about `image doesn't exists`.


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gramineproject/gsc/183)
